selfdrive/controls/lib/latcontrol_pid.py

```diff
@@ -18,6 +19,8 @@ def update(self, active, v_ego, angle_steers, angle_steers_rate, steer_override,
     pid_log = log.ControlsState.LateralPIDState.new_message()
     pid_log.steerAngle = float(angle_steers)
     pid_log.steerRate = float(angle_steers_rate)
+    driver_opposing = steer_override and (angle_steers - self.prev_angle_steers) * self.pid.i < 0
```

Comment:
mmm... I would check for driver torque being of opposite sign wrt torque applied. I get the idea of your change though, I can take it from here if you like.

Reply:
I checked all of the interfaces for any vehicles with a negative value for SteerKiV, but they were all positive.  It seems that a negative SteerKiV would be the only problem for the logic above.